app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+from typing import List
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
```

Comment:
You forgot to add a file extension, for example cinema.db3

Reply:
I didn't forgot it without file extension)